### PR TITLE
FIX, VIZ: ensure figs appear

### LINKED ISSUE #8441: Raw.plot() issue
I found something maybe a bug.
Now I used the master branch in GitHub.  I use Pycharm and terminal to run codes. One of my examples uses **raw.plot()** to plot raw data.  While I ran codes in Pycharm's console, **raw.plot()** went well, a figure showed up. While I ran .py in terminal like **python test.py**, no figure showed up. And if I type **python** in the terminal and type the codes in this way, also no figures showed up. But if I used the stable mne, everything was OK.

### PR DESCRIPTION
**Pull request by drammock**
closes #8441

### CODE REVIEW

**Comment by drammock** [comment]
@hoechenberger after this is merged and #8381 is rebased, this should also fix the issue you mentioned about the help button not working.

**Review by hoechenberger — APPROVED** [review]

File: mne/viz/_figure.py [hunk 1]
```diff
@@ -1173,6 +1175,7 @@ def _create_selection_fig(self):
         # add event listeners
         radio_ax.buttons.on_clicked(fig._radiopress)
         fig.canvas.mpl_connect('lasso_event', fig._set_custom_selection)
+        #plt_show(fig=fig)
```
Comment (agramfort) [inline mne/viz/_figure.py:1178]: cleanup?

File: mne/viz/_figure.py [hunk 2]
```diff
@@ -695,6 +695,7 @@ def _create_ch_location_fig(self, idx):
         # fig.lasso.alpha_other = 0.3
         # fig.lasso.linewidth_selected = 3
         # fig.lasso.style_sensors(inds)
+        # plt_show(fig=fig)
```
Comment (agramfort) [inline mne/viz/_figure.py:698]: cleanup?

Comment (hoechenberger) [reply mne/viz/_figure.py:698]: I think these lines were left here on purpose…

File: mne/viz/_figure.py (hunk 1 above)
Comment (hoechenberger) [reply mne/viz/_figure.py:1178]: this one ideed looks like cruft :)

File: mne/viz/_figure.py (hunk 2 above)
Comment (drammock) [reply mne/viz/_figure.py:698]: indeed, those are dealt with in #8381

**Review by agramfort — APPROVED** [review]

**Pull request merged**